app/graphs.py

```diff
@@ -1,14 +1,17 @@
-"""The following file is dedicated to storing graphs functions.
+'''
```

Comment:
Hey @pechouc, thank you for this PR ! I agree with everthing. Once we merged this PR to master, we can change the calls to geojson departements and the historic fires, so as discussed in #35 

I just have a small question: for the docstring, should we use """ or ''' ? I always use """ on my end as it's described [here](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings). But we can use ''' if it's what have been chosen. However I think that we need to stay consistent throughout the repo and here I can see that we are using """ in some docstrings (for instance in `build_info_object`) and ''' in others. What do you think ?

Reply:
To be honest, the `'''` option has not been explicitly chosen and I did not know that there was a proper way to write these docstrings. I agree that we should stay consistent throughout the repository and for the sake of rigour, I will make the switch to `""""` in a few hours. Thanks for the heads-up!

Reply:
Good for me 🤗

Reply:
It seems that this solved the Codacy issue, nice catch!